test2_2/list.c

```diff
@@ -0,0 +1,116 @@
+#include <stdlib.h>
+#include <stdio.h>
+
+#include "list.h"
+
+typedef struct SortedListElement {
+    int value;
+    struct SortedListElement *next;
+} SortedListElement;
+
+typedef struct SortedList {
+    SortedListElement *next;

```

Comment:
```suggestion
    SortedListElement *head; // or first
```
Почему первый элемент в списке называется "следующим"?